### PR TITLE
Updated broken link on Joy of Coding Page

### PR DESCRIPTION
**Pull request by elebumm**
## Description
There is a broken link on the Joy of Coding Page. I have replaced it with the official Youtube Playlist.

### CODE REVIEW

**Comment by pmac** [comment]
This looks good to me! Thanks for the patch! I just have 2 questions:

1. Is this the official play list? Perhaps we should ask Mike Conley?
2. Is changing this string okay or do we need an l10n tag? I think this page might be decently well translated.

I'll check on (2) in a bit.

**Comment by pmac** [comment]
Answer to (2) is: it is not translated so we can change that string at will. But I think we should still check with mconley about the proper link.

**Comment by mikeconley** [comment]
The YouTube page is probably fine, but perhaps Andy Kochendorfer knows how we might link directly to the playlist on the new Air Mozilla so that anonymous users can access it. I'll point him at this issue.

**Comment by elebumm** [comment]
Thanks guys!

File: bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html [hunk 1]
```diff
@@ -24,8 +24,8 @@ <h3>{{ _('Watch the intro.') }}</h3>
         </video>
         <button type="button" class="button-blue watch" data-action="play" data-task="joyofcoding" data-step="one" data-complete="true">{{ _('Watch the video') }}</button>
         {% if l10n_has_tag('contribute-tasks-0516') %}
-          <p><a class="more" rel="external" href="https://air.mozilla.org/channels/livehacking/">
-            {{ _('View more <em>Joy of Coding</em> videos on Air Mozilla') }}
+          <p><a class="more" rel="external" href="https://www.youtube.com/playlist?list=PLmaFLMwlbk8wKMvfEEzp9Hfdlid8VYpL5">
+            {{ _('View more <em>Joy of Coding</em> videos.') }}
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html]: This link has a `»` added to it by the CSS so the addition of the period looks a little funny.

**Comment by stephaniehobson** [comment]
This is a good change, let's get it into production! We can update the link to AirMo later if we get a better URL.

**Comment by mikeconley** [comment]
Any updates on this? [A Twitter user reached out today to let me know about the broken link](https://twitter.com/RangTiger/status/1052220712163905536).

**Comment by elebumm** [comment]
I am unsure as to who this has to be passed along to have it submitted. Any ideas?

File: bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html]:
```suggestion
            {{ _('View more <em>Joy of Coding</em> videos') }}
```

**Comment by stephaniehobson** [comment]
Could you please remove the period and then we will get this merged :)

**Comment by elebumm** [comment]
Done!

**Pull request merged**